src/Admin/SupportMenu.php

```diff
@@ -0,0 +1,250 @@
+		$this->google_fonts = $google_fonts;
+	}
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		/**
+		 * Filter whether to enable the AMP settings.
+		 *
+		 * @param bool $enable Whether to enable the AMP settings. Default true.
+		 *
+		 * @since 0.5
+		 */
+		return (bool) apply_filters( 'amp_support_menu_is_enabled', true );
+	}
+
+	/**
+	 * Adds hooks.
+	 */
+	public function register() {
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+		add_action( 'admin_menu', [ $this, 'add_menu_items' ], 9 );
+		add_action( 'wp_ajax_' . self::AJAX_ACTION, [ $this, 'ajax_callback' ] );
+
+	}
+
+	/**
+	 * Returns the slug for the support page.
+	 *
+	 * @return string
+	 */
+	public function get_menu_slug() {
+
+		return 'amp-support';
+	}
+
+	/**
+	 * Provides the support screen handle.
+	 *
+	 * @return string
+	 */
+	public function screen_handle() {
+
+		return sprintf( 'amp_page_%s', $this->get_menu_slug() );
+	}
+
+	/**
+	 * Add menu.
+	 */
+	public function add_menu_items() {
+
+		require_once ABSPATH . '/wp-admin/includes/plugin.php';
+
+		add_submenu_page(
+			$this->parent_menu_slug,
+			esc_html__( 'Support', 'amp' ),
+			esc_html__( 'Support', 'amp' ),
+			'manage_options',
+			$this->get_menu_slug(),
+			[ $this, 'render_screen' ],
+			10
+		);
+
+	}
+
+	/**
+	 * Enqueues settings page assets.
+	 *
+	 * @param string $hook_suffix The current admin page.
+	 *
+	 * @return void
+	 */
+	public function enqueue_assets( $hook_suffix ) {
+
+		if ( $this->screen_handle() !== $hook_suffix ) {
+			return;
+		}
+
+		/** This action is documented in includes/class-amp-theme-support.php */
+		do_action( 'amp_register_polyfills' );
+
+		$asset_file   = AMP__DIR__ . '/assets/js/' . self::ASSET_HANDLE . '.asset.php';
+		$asset        = require $asset_file;
+		$dependencies = $asset['dependencies'];
+		$version      = $asset['version'];
+
+		wp_enqueue_script(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'js/' . self::ASSET_HANDLE . '.js' ),
+			$dependencies,
+			$version,
+			true
+		);
+
+		wp_enqueue_style(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'css/amp-support.css' ),
+			[
+				$this->google_fonts->get_handle(),
+				'wp-components',
+			],
+			AMP__VERSION
+		);
+
+		$args    = [];
+		$post_id = filter_input( INPUT_GET, 'post_id', FILTER_SANITIZE_NUMBER_INT );
+
+		if ( ! empty( $post_id ) && 0 < intval( $post_id ) ) {
+			$args = [
+				'amp_validated_post_ids' => [
+					$post_id,
+				],
+			];
+		}
+
+		$support_service = Services::get( 'support' );
+		$data            = $support_service->get_data( $args );
+
+		wp_localize_script(
+			self::ASSET_HANDLE,
+			'ampSupportData',
+			[
+				'action' => self::AJAX_ACTION,
+				'nonce'  => wp_create_nonce( self::AJAX_ACTION ),
+				'args'   => $args,
+				'data'   => $data,
+			]
+		);
```

Comment:
Using `wp_localize_script()` is not preferable because it casts all keys to strings.

It's better to use `wp_add_inline_script()`, like so:

https://github.com/ampproject/amp-wp/blob/c23b3b0df63390f9cbacd8ec7f242ac513a10159/src/Admin/OnboardingWizardSubmenuPage.php#L248-L255